junebug/channel.py

```diff
@@ -151,19 +162,17 @@ def get_all(cls, redis):
         channels = yield redis.smembers('channels')
         returnValue(channels)
 
-    @classmethod
     @inlineCallbacks
-    def send_message(cls, id, sender, outbounds, msg):
+    def send_message(self, id, sender, outbounds, msg):
         '''Sends a message.'''
         event_url = msg.get('event_url')
         msg = message_from_api(id, msg)
         msg = TransportUserMessage.send(**msg)
-        msg = yield cls._send_message(id, sender, outbounds, event_url, msg)
+        msg = yield self._send_message(id, sender, outbounds, event_url, msg)
         returnValue(api_from_message(msg))
```

Comment:
I think we don't need `id` anymore know that we have that as an attribute on the instance.